xwiki-platform-core/xwiki-platform-user/xwiki-platform-user-profile/xwiki-platform-user-profile-test/xwiki-platform-user-profile-test-docker/pom.xml

```diff
@@ -44,12 +44,6 @@
       <version>${project.version}</version>
       <type>xar</type>
     </dependency>
-    <dependency>
```

Comment:
You sure about this change ? Does not seems to have anything to do with this issue.